### DISCUSSION TITLE
Lost i'm a new world

### DISCUSSION THREAD

**Discussion by SamySalamy87x** [community/community#143183] [New to GitHub] [answered]
I'm new to GitHub and the whole world of programming, AI, and project development. I come from a background in music production and surgical assistance, and while I have a lot of experience in those areas, navigating GitHub and developing an AI project is entirely new territory for me.

Currently, I’m working on a healthcare project that combines AI and blockchain to improve medical services, and I feel a bit lost with all the technical aspects. I would greatly appreciate any guidance, support, or advice from anyone who has traveled a similar path or has experience in these fields.

Additionally, I already have the Digital Dr GPT and Abril Mental Helper GPT ready. Digital Dr is designed to provide a virtual pre-consultation in emergencies, especially if you find yourself in a situation where visiting a doctor isn’t possible. On the other hand, Abril Mental Helper GPT is a holistic psychology tool that can help you balance and manage difficult situations.

If anyone needs music for a project, feel free to use my skills as a music producer. I can also offer a creative music therapy session if you’re feeling stressed, or teach surgical techniques if you're interested. I'm open to exchanging this knowledge in return for some advice to help me reach my goals.

All four options are available on the page I’ve just set up on my GitHub profile. I look forward to learning from this community and contributing however I can. Thank you for any help you can offer!

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @SamySalamy87x 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by SamySalamy87x** [reply]
Thank you very much, I'm going to check out what you said bro, if you need a song I'll gladly make it for you

**Comment by CyberSphinxxx** [comment]
Welcome to GitHub! Your mix of healthcare, AI, and music production sounds amazing, and your projects could make a real impact. Diving into all these new tools can feel overwhelming, but you’ve got a unique perspective that’s bound to bring something special. Keep at it, this community is here to help, and I’m sure you’ll find lots of support. Good luck!

**Reply by SamySalamy87x** [reply]
Thank you very much cyber here we are to serve you

**Comment by AuroraEchos** [comment, marked as answer]
Hi bro, welcome to the community🥳, I was just like you at the beginning, and I want to share three lessons with you: keep practicing, ask for help, and stay motivated. Come on bro. 💪

**Reply by SamySalamy87x** [reply]
Hello, thank you very much, I really appreciate it. I have been trying to integrate and create more new things like Redis, which is another world too, haha. But thank you very much!!!! I really feel great!

**Comment by mgunawardhana** [comment]
> I'm new to GitHub and the whole world of programming, AI, and project development. I come from a background in music production and surgical assistance, and while I have a lot of experience in those areas, navigating GitHub and developing an AI project is entirely new territory for me.
> 
> Currently, I’m working on a healthcare project that combines AI and blockchain to improve medical services, and I feel a bit lost with all the technical aspects. I would greatly appreciate any guidance, support, or advice from anyone who has traveled a similar path or has experience in these fields.
> 
> Additionally, I already have the Digital Dr GPT and Abril Mental Helper GPT ready. Digital Dr is designed to provide a virtual pre-consultation in emergencies, especially if you find yourself in a situation where visiting a doctor isn’t possible. On the other hand, Abril Mental Helper GPT is a holistic psychology tool that can help you balance and manage difficult situations.
> 
> If anyone needs music for a project, feel free to use my skills as a music producer. I can also offer a creative music therapy session if you’re feeling stressed, or teach surgical techniques if you're interested. I'm open to exchanging this knowledge in return for some advice to help me reach my goals.
> 
> All four options are available on the page I’ve just set up on my GitHub profile. I look forward to learning from this community and contributing however I can. Thank you for any help you can offer!

Hey there, and welcome to GitHub and the world of programming! 👋 It’s super inspiring to see someone from a music and surgical background jumping into AI and blockchain for healthcare! 💻💉

Starting out can feel like a lot 🌊, but don’t worry—everyone feels that way at first! Here are a few tips to help you get comfortable:

1. **Get to Know GitHub** 🛠️: GitHub might feel complex, but the basics (like repos, commits, and branches) are easier than they seem! Try GitHub’s Learning Lab or check out some beginner tutorials on freeCodeCamp. 

2. **AI & Blockchain Resources** 🧠🔗: For AI, libraries like PyTorch and TensorFlow are your friends! And for blockchain, check out Ethereum and Solidity to get into smart contracts. 

3. **Organize Your Project** 📂✨: With your awesome project ideas (Digital Dr GPT and Abril Mental Helper GPT), getting organized early will help a ton. Simple things like a good README file and tools like Docker for easy deployment can make life easier.

4. **Connect with the Community** 🌍💬: The open-source community is full of friendly folks. Don’t be afraid to ask questions on Stack Overflow or join places like Reddit’s r/learnprogramming. There’s tons of support for beginners!

Also, your offer to help with music production, therapy, and surgical techniques is amazing! 🎶🧘‍♂️🩺 That’s such a unique way to connect and give back.

If you need a hand with anything specific, feel free to reach out. We’re all here to help! Welcome again, and good luck with your journey! 🚀

**Reply by SamySalamy87x** [reply]
Hi Mark, thanks for taking the time to leave this valuable information. I was checking out micropython and I came up with an idea for a micropacemaker in the future. But that's a different world. That information you gave me is great, thank you. If you're around here, send me a message, okay? Do you know how to use Redis?

**Comment by ghostinhershell** [comment]
Just wanted to circle back and say that I love to see all of the encouraging words being shared in this post. Folks like you are what make this community a community! #PullTogether #CommitToCommunity #GitHub 🚀

**Reply by SamySalamy87x** [reply]
Hi ghost, good name, I thought I had answered you but I already saw that I didn't. Thank you very much. It really is a very cool community, they make you feel like a super newbie. But everyone is also super cool. 

I hope one day to make a song about this as the project develops.
